### ISSUE TITLE
Metrics for Prometheus are no longer returned when polling devices

### ISSUE THREAD

**Issue by valerarar** [czerwonk/junos_exporter#177]
Metrics for Prometheus are no longer returned when polling devices a junos_exporter 
This happens even when polling 10 devices
As a result, the alerts will no longer fire:
![image](https://user-images.githubusercontent.com/46557544/158762915-75045773-0815-4ffa-8ddc-9d95ee56a1f2.png)

Is it possible to store data statically in a file permanently and update as new values come in?

**Comment by lethalwp** [comment]
I haven't checked this particular issue. But is it possible that your polling rate is too low? So prometheus "forgets" about the metric.  If it is this problem, the solution is either to poll more frequently,  or change a prometheus option, i think it is: --query.lookback-delta=15m  which i've set at 15minutes for my usage case

**Comment by valerarar** [comment]
No, the problem is that while the metrics are being collected, the current metrics become unavailable. And when the alert manager polls the value, it gets no data.

**Comment by ouafnico** [comment]
I have exactly the same issue.
